config/locales/global_activities/en.yml

```diff
@@ -104,10 +104,12 @@ en:
       create_column_inventory_html: "%{user} created column %{repository_column} in inventory %{repository}"
       edit_column_inventory_html: "%{user} edited column %{repository_column} in inventory %{repository}"
       delete_column_inventory_html: "%{user} deleted column %{repository_column} in inventory %{repository}"
+      copy_inventory_html: "%{user} copied inventory %{repository_new} from inventory %{repository_original} as template"
+      copy_inventory_item_html: "%{user} copied inventory item %{repository_row_new} from inventory item %{repository_row_original}"
       create_protocol_in_repository_html: "%{user} created protocol %{protocol} in Protocol repository"
-      add_step_to_protocol_repository_html: "%{user} created protocol %{protocol}'s step %{step} in Protocol repository"
-      edit_step_in_protocol_repository_html: "%{user} edited protocol %{protocol}'s step %{step} in Protocol repository"
-      delete_step_in_protocol_repository_html: "%{user} deleted protocol %{protocol}'s step %{step} in Protocol repository"
+      add_step_to_protocol_repository_html: "%{user} created protocol %{protocol}'s step %{step_position} %{step} in Protocol repository"
```

Comment:
Please check this PR, it changes some of these strings https://github.com/biosistemika/scinote-web/pull/1616

Reply:
👍